docs/installation/arch.rst

```diff
@@ -1,20 +1,19 @@
 .. _arch-install:
 
-****************************
-Arch Linux: Install from AUR
-****************************
+**************************************
+Arch Linux: Install from ``[community]``
+**************************************
 
 If you are running Arch Linux, you can install Mopidy using the
-`mopidy <https://aur.archlinux.org/packages/mopidy/>`_ package found in AUR.
+`mopidy <https://aur.archlinux.org/packages/mopidy/>`_ package found in ``[community]``.
```

Comment:
Link is outdated. Correct is https://www.archlinux.org/packages/community/any/mopidy/

Same comment about "[community]" again.